Tribes/src/app/constants.ts

```diff
@@ -1,3 +1,5 @@
+import { HttpHeaders } from '@angular/common/http';
```

Comment:
ezt miert?

Reply:
Passz. De azóta már nincs is ott, kivenni sem tudom. :)